README.md

```diff
@@ -1,51 +1,116 @@
 
-## Table of Contents
+## Development
 
-* [Getting Started](#getting-started)
-* [Usage Instructions](#usage-instructions)
-    * [General Info](#general-info)
-    * [Moab to Catalog](#m2c) (M2C) existence/version check
-    * [Catalog to Moab](#c2m) (C2M) existence/version check
-    * [Checksum Validation](#cv) (CV)
-    * [Seed the catalog](#seed-the-catalog-with-data-about-the-moabs-on-the-storage-roots-the-catalog-tracks----presumes-rake-dbseed-already-performed)
-    * [Update the Catalog Because a Moab Moved](#migrate_moab_manually)
-* [Development](#development)
-    * [Running tests](#running-tests)
-    * [Dockerized Development](#docker)
-* [Deploying](#deploying)
-    * [Resque Pool](#resque-pool)
-* [API](#api)
-    * [Authentication/Authorization](#authn)
-    * [V1](#v1)
+Here are the steps you need to follow to get a working development instance of *preservation_catalog* running:
 
-## Getting Started
+### Install dependencies
 
-### Installing dependencies
+First get the Ruby dependencies:
+```sh
+bundle install
+```
 
 Use `docker compose` to start supporting services (PostgreSQL and Redis)
 ```sh
 docker compose up -d db redis
 ```
 
-### Configuring The database (ensure all defined storage roots, cloud endpoints, etc have the necessary DB records)
+### Configure the database 
+
+Set up the database:
 
-Run this script:
 ```sh
 ./bin/rails db:reset
+```
+
+Ensure storage roots and cloud endpoints have the necessary database records:
+
+```sh
 RAILS_ENV=test ./bin/rails db:seed
 ```
 
-# Usage Instructions
+### Run Tests
+
+To run the tests:
+
+```sh
+bundle exec rspec
+```
+
+### Run the App
+
+If you've chosen to install Ruby dependencies you can run the app locally with:
+
+```sh
+bin/dev
+```
+
+If you prefer you can also run the web app inside of a Docker container:
+
+```sh
+docker compose build app
+```
+
+Bring up the docker container and its dependencies:
+
+```sh
+docker compose up -d
+```
+
+It does the same thing as the `db:reset` and `db:seed` above, but you can
+initialize the database directly from inside in the container if you like:
+
+```sh
+docker compose run app bundle exec rails db:reset db:seed
+```
+
+All set, now you can view the app:
+
+Visit `http://localhost:3000/dashboard/`
+
+### Interact with API
+
+You can use curl to interact with the API via localhost:
+
+```sh
+curl -H 'Authorization: Bearer eyJhbGcxxxxx.eyJzdWIxxxxx.lWMJ66Wxx-xx' -F 'druid=druid:bj102hs9688' -F 'incoming_version=3' -F 'incoming_size=2070039' -F 'storage_location=spec/fixtures/storage_root01' -F 'checksums_validated=true' http://localhost:3000/v1/catalog
+```
+
+```sh
+curl -H 'Authorization: Bearer eyJhbGcxxxxx.eyJzdWIxxxxx.lWMJ66Wxx-xx' http://localhost:3000/v1/objects/druid:bj102hs9688
+
+{
+  "id":1,
+  "druid":"bj102hs9688",
+  "current_version":3,
+  "created_at":"2019-12-20T15:04:56.854Z",
+  "updated_at":"2019-12-20T15:04:56.854Z",
+  "preservation_policy_id":1
+}
+```
+
+### Publish Image
```

Comment:
Why do we publish the image?

Reply:
I wondered the same. Should this be removed?

Reply:
+1

Reply:
I've removed the information about publishing the image since this is something that CircleCI does for us anyway.